schema/codeception.schema.yml

```diff
@@ -0,0 +1,231 @@
+$id: 'https://codeception.com/1/codeception.schema.yml'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+title: 'codeception.dist.yml'
+description: 'Schema definition for Codeception main configuration YAML file.'
+
+definitions:
+    actor_suffix:
+        type: string
+        default: 'Tester'
+
+    namespace:
+        type: string
+        default: ''
+
+    include:
+        type: array
+        default: []
+
+    paths:
+        type: object
+        additionalProperties:
+            type: string
+        properties:
+            test:
+                type: string
+            log:
+                type: string
+            data:
+                type: string
+            support:
+                type: string
+
+    extends:
+        # @todo
+        oneOf:
+            -
+                type: 'null'
```

Comment:
I'm pretty sure that it doesn't make sense to set optional parameters to null in yaml file.

Reply:
The default value is `null`.
https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Configuration.php#L116

Reply:
Null is the default value in the PHP code, it isn't necessary to set settings to null in yaml file and it can be forbidden.
A more usual approach to emit unnecessary settings.